client/signup/main.jsx

```diff
@@ -745,6 +746,7 @@ class Signup extends React.Component {
 					/>
 				) }
 				<div className="signup__steps">{ this.renderCurrentStep() }</div>
+				<QueryExperiments />
```

Comment:
You should rather be using `<Experiment name="experimen_name" />`.

See this guide for more info: https://github.com/Automattic/experimentation-platform/wiki/Migration-guide-from-%60calypso-abtest%60#accessing-explat-state-directly

Reply:
I'm using the `<Experiment />` component in client/signup/steps/plans/index.jsx, so this is redundant. I've now removed this line.